templates/automotive/index.html

```diff
@@ -323,236 +403,332 @@ <h2 class="p-heading--5">Over-the-air updates</h2>
+      <div class="p-logo-section__items u-no-margin--bottom">
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/1f9d6318-lyft-logo.png",
+            alt="",
+            width="288",
+            height="288",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/5eaf9fae-cruise-logo.png",
+            alt="",
+            width="288",
+            height="288",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/e2608cee-zoox-logo.png",
+            alt="",
+            width="288",
+            height="288",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/eadcdf0b-outrider-logo.png",
+            alt="",
+            width="288",
+            height="288",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+      </div>
     </div>
   </div>
 </section>
-
 <section class="p-strip--light">
-  <div class="row">
-    <div class="col-6">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/8dd99b80-ubuntu-logo14.png",
-        alt="",
-        height="100",
-        width="223",
-        hi_def=True,
-        ) | safe
-      }}
-      <h2 class="p-heading--5">Software strategy acceleration</h2>
-      <p class="p-heading--2">Trusted by executives. Loved by gearheads.</p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">The <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">best developers choose Ubuntu</a>.</li>
-        <li class="p-list__item is-ticked">The biggest pool of talent. The broadest range of tools.  The community that counts.</li>
-        <li class="p-list__item is-ticked">One company stands behind it all &mdash; Canonical.</li>
-      </ul>
+  <div class="u-fixed-width">
+    <h2>Preferred by developers</h2>
+    <p>Ubuntu is the <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">preferred OS for developers</a>, at 66% of working professionals &ndash; and growing.</p>
+    <p>Empower your team and attract the best talent.</p>
+    <p><a href="/automotive/developer">Read more about the Ubuntu developer experience&nbsp;&rsaquo;</a></p>
```

Comment:
```suggestion
    <p><a href="/automotive/developers">Read more about the Ubuntu developer experience&nbsp;&rsaquo;</a></p>
```
Will this /automative/developers be live before this is merged, if not we should remove this link for now